src/components/Forms/Login/LoginForm.tsx

```diff
@@ -54,7 +54,12 @@ function LoginForm({ onSubmit, initialValues, clearErrors, validate }: Props) {
 		>
 			{(formik) => {
 				return (
-					<Form className={classes.root} autoComplete="off" data-testid="form">
+					<Form
+						className={classes.root}
+						autoComplete="off"
```

Comment:
why auto-complete off?

Reply:
This was there before i made changes to the component

Reply:
Ok